assets/js/amp-invalid-url-post-edit-screen.js

```diff
@@ -56,5 +62,153 @@ var ampInvalidUrlPostEditScreen = ( function() { // eslint-disable-line no-unuse
 		return component.data.l10n.unsaved_changes;
 	};
 
+	/**
+	 * Updates the <tr> with 'Showing x of y validation errors' at the top of the list table with the current count.
+	 * If this does not exist yet, it creates the element.
+	 *
+	 * @param {number} numberErrorsDisplaying - The number of errors displaying.
+	 */
+	component.updateShowingErrorsRow = function updateShowingErrorsRow( numberErrorsDisplaying = null ) {
+		var thead, tr, th,
+			theadQuery = document.getElementsByTagName( 'thead' ),
+			idNumberErrors = 'number-errors';
+
+		// Only create the <tr> if it does not exist yet.
+		if ( theadQuery[ 0 ] && ! document.getElementById( idNumberErrors ) ) {
+			thead = theadQuery[ 0 ];
+			tr = document.createElement( 'tr' );
+			th = document.createElement( 'th' );
+			th.setAttribute( 'id', idNumberErrors );
+
+			/* eslint-disable dot-notation */
+			th.style[ 'text-align' ] = 'center';
+			th.style[ 'background-color' ] = '#d3d3d3b8';
+			th.style[ 'color' ] = '#1e8cbecc';
+			/* eslint-enable dot-notation */
+			th.setAttribute( 'colspan', '6' );
+			tr.appendChild( th );
+			thead.appendChild( tr );
+		}
+
+		// Update the number of errors displaying.
+		if ( null !== numberErrorsDisplaying ) {
+			document.getElementById( idNumberErrors ).innerText = component.data.l10n.showing_number_errors.replace( '%', numberErrorsDisplaying );
+		}
+	};
+
+	/**
+	 * Handles filtering by error type, triggered by clicking 'Apply Filter'.
+	 *
+	 * Gets the value of the error type <select> element.
+	 * And hides all <tr> elements that do not have the same type of this value.
+	 * If 'All Error Types' is selected, this displays all errors.
+	 */
+	component.handleFiltering = function handleFiltering() {
+		var onChange = function( event ) {
+			var numberErrorsDisplaying = 0;
+
+			if ( ! event.target.matches( 'select' ) ) {
+				return;
+			}
+
+			event.preventDefault();
+
+			/*
+			 * Iterate through all of the <tr> elements in the list table.
+			 * If the error type does not match the value (selected error type), hide them.
+			 */
+			document.querySelectorAll( '[data-error-type]' ).forEach( function( element ) {
+				var errorType = element.getAttribute( 'data-error-type' );
+
+				// If the value is '-1', 'All Error Types' was selected, and this should display all errors.
+				if ( event.target.value === errorType || '-1' === event.target.value ) {
+					element.parentElement.parentElement.classList.remove( 'hidden' );
+					numberErrorsDisplaying++;
+				} else {
+					element.parentElement.parentElement.classList.add( 'hidden' );
+				}
+			} );
+
+			component.updateShowingErrorsRow( numberErrorsDisplaying );
+		};
+
+		document.getElementById( 'amp_validation_error_type' ).addEventListener( 'change', onChange );
+	};
+
+	/**
+	 * Handles searching for errors via the <input> and the 'Search Errors' <button>.
+	 */
+	component.handleSearching = function handleSearching() {
```

Comment:
This is a very basic search of the 'Details' column when clicking 'Search Errors.' 

* It only searches the 'Details' column
* If you search something like `JavaScript`, it won't find `javascript`

Maybe I should improve this.